readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -55,12 +56,18 @@ function attach_elastic_search_query(data) {
 
                         // Show highlighted texts
                         if (highlight.content) {
-                            var content_text = xss(highlight.content[0]);
-                            var contents = $('<div class="context">');
-
-                            contents.html(content_text);
-                            contents.find('em').addClass('highlighted');
-                            list_item.append(contents);
+                            for (index in highlight.content) {
```

Comment:
So, JS quirk here. `for ... in` isn't actually for iterables, it's for properties on an object. It's better to use the old `for (var i; ...)` approach for arrays:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

There is `for ... of`, which loops over iterables, but browser support is still new:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Browser_compatibility